paasta_tools/kubernetes_tools.py

```diff
@@ -2601,13 +2601,19 @@ def sanitize_for_config_hash(
         if ahash["metadata"] is not None:
             ahash["metadata"]["namespace"] = None
 
-        # soa-configs SHA
+        # env vars to exclude
         try:
             for container in ahash["spec"]["template"]["spec"]["containers"]:
                 container["env"] = [
                     e
                     for e in container["env"]
-                    if e.get("name", "") != "PAASTA_SOA_CONFIGS_SHA"
+                    if e.get("name", "")
+                    not in (
+                        "PAASTA_SOA_CONFIGS_SHA",
+                        # TODO: here temporarily to avoid a restart of all services
+                        # at the same time, remove once things have settled
+                        "AWS_SDK_UA_APP_ID",
```

Comment:
fwiw, i think we'll still big-bounce when this is removed from here since hashing the dict with and without this will result in different hashes, no?

Reply:
Ah, shoot, you are right, cause the job reconciling deployments just looks at the hash in the labels of the deployment, right? Rather than re-hashing the whole k8s config.

Welp, then I have no painless plan, if not just leave it excluded forever.

Reply:
tron big bounces are a little more painful due to the config update endpoint being pretty slow, but hopefully my recent changes make that a little bit less painful

if anything, it'd be nice to roll this out for just paasta OR tron, and once things have settled, rollout for the other workload

Reply:
I can adjust for that.